packages/ui5-tooling-modules/lib/templates/jsdoc/ClassHeader.hbs

```diff
@@ -1,6 +1,10 @@
 /**
  * @class
  * {{{description}}}
+{{#if interfacesSlashed}}
```

Comment:
to avoid empty lines in the output, please move the conditional handlebars tags into the previous line, e.g.:

```
/**
 * @class
 * {{{description}}}{{#if interfacesSlashed}}
 * @implements {{{interfacesSlashed}}}{{/if}}
 * {{{implementsTag}}}
 * @extends {{{extendsTag}}}
 * @constructor
 * @public
```